File: code/intermediate/polymorphism/methodOverloading/MethodOverloading.py
def print_name(*args):
    if len(args) == 1:
        print("Name :-" + args[0])
    elif len(args) == 2:
        print("Name :-" + args[0], "Age :-", + args[1])
    elif len(args) == 3:
        print("Name :-" + args[0], "Age :-", + args[1], "Height :-", +args[2])

print_name("Name")
print_name("Name", 24)
print_name("Name", 24, 5.6)

File: code/intermediate/polymorphism/methodOverloading/MethodOverloading.js
function printName() {
    if (arguments.length === 1) {
        console.log("Name :-", arguments[0]);
    } else if (arguments.length === 2) {
        console.log("Name :-", arguments[0], "Age", arguments[1]);
    } else if (arguments.length === 3) {
        console.log("Name :-", arguments[0], "Age", arguments[1], 'Height', arguments[2]);
    } else {
        throw Error("Invalid number of arguments");
    }
}

printName('Avinash');
printName('Avinash', 23);
printName('Avinash', 23, 5.6);